src/components/blog/Blog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Grid} from '@material-ui/core';
```

Comment:
if you import only one element you can import from directory @material-ui/core/Grid